src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -164,19 +170,38 @@ export class EuiSelectableListItem extends Component<
       instruction = (
         <EuiI18n
           token="euiSelectableListItem.excludedOptionInstructions"
-          default="To uncheck this option, press enter."
+          default="To uncheck this option, press Enter."
+        />
+      );
+    } else if (allowExclusions && checked === 'mixed') {
+      state = (
+        <EuiI18n
+          token="euiSelectableListItem.excludedOption"
+          default="Mixed (indeterminate) option."
+        />
```

Comment:
CI is failing because of this token BTW. You can't reuse the same token for different strings, this one needs its own token ID.
```suggestion
        <EuiI18n
          token="euiSelectableListItem.mixedOption"
          default="Mixed (indeterminate) option."
        />
```